journal-writings/modernized/06-chapter-4.adoc

```diff
@@ -52,9 +52,7 @@ and be so supported as to remain unmoved at the faces of men.
 
 As it is common for Friends on such a visit to have entertainment free of cost,
 a difficulty arose in my mind with respect to saving my money by kindness received,
-which to me appeared to be the gain of oppression.
-
-Receiving a gift, considered as a gift,
+which to me appeared to be the gain of oppression. Receiving a gift, considered as a gift,
```

Comment:
The general rule is: keep the beginning of a sentence on a new line. So I would _not_ have pulled up the `Receiving a gift...` onto the same line.